### DISCUSSION TITLE
Type issue in Prisma query

### DISCUSSION THREAD

**Discussion by hashika2** [prisma/prisma#18418] [Q&A]
### Bug description

When I write a SQL query inside the prisma method called prisma.$queryRawUnsafe. It's getting run time error 
`[Nest] 24348   - 03/22/2023, 3:39:58 PM   [ExceptionsHandler] Do not know how to serialize a BigInt +5681ms
TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)`

also, the query looks like,
this.prisma.$queryRawUnsafe( 
`SELECT LocationDrop.*, COUNT(*) AS collectionCount FROM Manifest 
INNER JOIN Location AS LocationDrop ON Manifest.locationDropId = LocationDrop.id 
WHERE Manifest.deletedAt IS NULL  AND Manifest.type = "COLLECTION" AND Manifest.runSheetLineId =  ? 
GROUP BY Manifest.locationDropId`, runSheetLineWhereUniqueInput.id);

Also, I try to use prisma.queryRow as well. but the same issue.

### Expected behavior

The query should run without any error.

### Environment & setup

- Link to Data Browser instance: 
- OS: 
- Browser: 
- Database: 

### Screenshot of page where the problem happened (if applicable)

![prisma123](https://user-images.githubusercontent.com/37840298/226873601-a2127d76-5e2c-4ce6-8f06-822a4cfe86ca.PNG)

**Comment by janpio** [comment]
Your question is not related to Prisma Studio or Prisma Data Platform, Data Browser, is it? I moved the issue to a different repo and turned it into a discussion.

What do you do with the result of this query in your application? The error output indicates you are somehow JSON.stringify it in Nest?

**Reply by hashika2** [reply]
yes using nest

**Reply by janpio** [reply]
How exactly? What do you do with `summaryArray` after it contains the data?

Generally the solution to this problem is to add the ability to serialize BigInt: https://github.com/prisma/studio/issues/614#issuecomment-795213237

**Reply by hashika2** [reply]
just send to the font end

**Reply by janpio** [reply]
You have to give me more information than that if you want me to help you. 
Otherwise my answer has to be "Then just send it in a way that it does not cause this error".

**Reply by nurul3101** [reply]
@hashika2 Let us know in case you still need assistance. It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
